README.md

```diff
@@ -147,6 +147,17 @@ For example, the `workbenchConfigFilePath` field might have the `anyFolder/myCon
 This means that the config file can be found by the path: `che-code/branding/anyFolder/myConfigFiles/workbench-config.json`
 - Currently, the [branding.sh](https://github.com/che-incubator/che-code/blob/main/branding/branding.sh) script is not run automatically when building this project. It needs to be integrated into the build process of the [downstream branded project or product](https://github.com/redhat-developer/devspaces-images/blob/devspaces-3-rhel-8/devspaces-code/build/scripts/sync.sh#L96).
 
+# Builds
+
+This repo contains several [actions](https://github.com/che-incubator/che-code/actions), including:
+* ![release latest stable](https://github.com/che-incubator/che-code/actions/workflows/release.yml/badge.svg)
+* ![upstream rebase](https://github.com/che-incubator/che-code/actions/workflows/image-publish.yml/badge.svg)
+* ![PR publish](https://github.com/che-incubator/che-code/actions/workflows/pr-check-image-publish.yml/badge.svg)
```

Comment:
I see a svg icon when I click on a badge:

<img width="779" alt="image" src="https://user-images.githubusercontent.com/5676062/198501210-101f47c6-a5e1-4f9a-960f-733c72f81ff7.png">

Maybe it makes sense to use the following:
```
[![PR publish](https://github.com/che-incubator/che-code/actions/workflows/pr-check-image-publish.yml/badge.svg)](https://github.com/che-incubator/che-code/actions/workflows/pr-check-image-publish.yml)
```
It will allow to go directly to the corresponding action by click.

Reply:
damn, good catch. I didn't notice that when I was testing the md in vscode preview :(

https://github.com/che-incubator/chectl#builds definitely has links to the badges instead of the actions. Will redo the PRs on https://github.com/eclipse/che/issues/21790#issuecomment-1293761004 as well as this one.